### PR TITLE
editEvent function updated to work on FireStore

### PR DESCRIPTION
**Pull request by BautistaBertolami**
* Changed the function for editEvent to work with firestore
* Added a simple function that @HanielDiaz suggested, to not have to keep re-writing the converter method
* Changed createEvent to use the previously established function for consistency

### CODE REVIEW

File: functions/src/Services/Event.ts [hunk 1]
```diff
@@ -2,19 +2,26 @@ import { User } from "../Models/User";
 import { Event, eventConverter } from "../Models/Event";
 import { firestore } from 'firebase-admin';
 
+/**
+ * This function is used to easily handle retrieving the event collection with the converter
+ */
+function getEventCollection() {
+	return firestore().collection('events').withConverter(eventConverter);
+}
+
```
Comment (idelmr) [inline functions/src/Services/Event.ts:11]: Should be a `const` variable instead

File: functions/src/Services/Event.ts [hunk 2]
```diff
@@ -2,19 +2,26 @@ import { User } from "../Models/User";
...
 export const createEvent = (event: Event) => {
-    const eventCollection = firestore().collection('events').withConverter(eventConverter);
+	const eventCollection = getEventCollection();
 	eventCollection.add(event);
 };
```
Comment (idelmr) [inline functions/src/Services/Event.ts:16]: Can be one line if `getEventCollection` is a variable

File: functions/src/Services/Event.ts [hunk 3]
```diff
@@ -2,19 +2,26 @@ import { User } from "../Models/User";
...
 export const editEvent = (event: Event) => {
-	// App.database().ref(`events/${event.id}`).once("value", snapshot => {
-	// 	if (snapshot.val())
-	// 		App.database().ref(`committees/${snapshot.val().committee}/events/`).update({ [event.id]: null });
-	// })
-	// App.database().ref(`/events/${event.id}`).update(event);
-	// if (event.committee)
-	// 	App.database().ref(`/committees/${event.committee}/events/`).update({ [event.id]: true });
+	const eventCollection = getEventCollection();
+	eventCollection.doc(event.id).update({ ...event });
```
Comment (idelmr) [inline functions/src/Services/Event.ts:22]: Can also be one line

File: functions/src/Services/Event.ts [hunk 4]
```diff
@@ -2,19 +2,26 @@ import { User } from "../Models/User";
...
 export const createEvent = (event: Event) => {
-    const eventCollection = firestore().collection('events').withConverter(eventConverter);
+	const eventCollection = getEventCollection();
 	eventCollection.add(event);
 };
 
+/**
+ * This function currently does not work with the eventConverter, hence the need for the spread operator ({ ...event })
+ * 
+ * getEventCollection() was used for consistency with the rest of the event functions
```
Comment (idelmr) [inline functions/src/Services/Event.ts]: Unnecessary comment

File: functions/src/Services/Event.ts [hunk 5]
```diff
@@ -2,19 +2,26 @@ import { User } from "../Models/User";
...
+}
+
 export const createEvent = (event: Event) => {
-    const eventCollection = firestore().collection('events').withConverter(eventConverter);
+	const eventCollection = getEventCollection();
 	eventCollection.add(event);
 };
 
+/**
+ * This function currently does not work with the eventConverter, hence the need for the spread operator ({ ...event })
```
Comment (idelmr) [inline functions/src/Services/Event.ts]: You can be more specific
```suggestion
 * The update method does not work with the event converter, so `event` is unpacked and passed in as a separate object.
```

File: functions/src/Services/Event.ts [hunk 6]
```diff
@@ -2,19 +2,26 @@ import { User } from "../Models/User";
 import { Event, eventConverter } from "../Models/Event";
 import { firestore } from 'firebase-admin';
 
+/**
+ * This function is used to easily handle retrieving the event collection with the converter
```
Comment (idelmr) [inline functions/src/Services/Event.ts]:
```suggestion
 * Handles retrieving the event collection with the converter.
```

**Review by idelmr — CHANGES_REQUESTED** [review]
* Minor comment suggestions
* Unclear about the need for `getEventCollection` to be a function

File: functions/src/Services/Event.ts (hunk 1 above)
Comment (HanielDiaz) [reply functions/src/Services/Event.ts:11]: why?
I think it should be a function

File: functions/src/Services/Event.ts (hunk 2 above)
Comment (HanielDiaz) [reply functions/src/Services/Event.ts:16]: did this for consistency with other functions

File: functions/src/Services/Event.ts (hunk 3 above)
Comment (HanielDiaz) [reply functions/src/Services/Event.ts:22]: stop with your manic one liners

File: functions/src/Services/Event.ts (hunk 5 above)
Comment (HanielDiaz) [reply functions/src/Services/Event.ts]: It should instead say that the event converter only works with set and get, so event is unpacked and passed in as a separate object in the update method.

File: functions/src/Services/Event.ts (hunk 1 above)
Comment (idelmr) [reply functions/src/Services/Event.ts:11]: Have fun testing more stuff

Comment (BautistaBertolami) [reply functions/src/Services/Event.ts:11]: I'll keep it as a function, we can always change it after

File: functions/src/Services/Event.ts (hunk 2 above)
Comment (BautistaBertolami) [reply functions/src/Services/Event.ts:16]: Yeah consistency, also sorry if I spam your email, I'm writing comments on all the changes to make sure I did not forget any

File: functions/src/Services/Event.ts (hunk 3 above)
Comment (BautistaBertolami) [reply functions/src/Services/Event.ts:22]: I'll keep it at two lines

File: functions/src/Services/Event.ts (hunk 4 above)
Comment (BautistaBertolami) [reply functions/src/Services/Event.ts]: this is an unnecessary comment

File: functions/src/Services/Event.ts (hunk 6 above)
Comment (BautistaBertolami) [reply functions/src/Services/Event.ts]: ok

File: functions/src/Services/Event.ts (hunk 5 above)
Comment (BautistaBertolami) [reply functions/src/Services/Event.ts]: Sounds good boss

**Review by idelmr — APPROVED** [review]

**Pull request merged**